worlds/aquaria/Items.py

```diff
@@ -0,0 +1,187 @@
+"""
+Author: Louis M
+Date: Fri, 15 Mar 2024 18:41:40 +0000
+Description: Manage items in the Aquaria game multiworld randomizer
+"""
+
+from typing import Optional
+from enum import Enum
+from BaseClasses import Item, ItemClassification
+
+class ItemType(Enum):
+    """
+    Used to indicate to the multi-world if an item is usefull or not
+    """
+    NORMAL = 0
+    PROGRESSION = 1
+    JUNK = 2
+
+class ItemGroup(Enum):
+    """
+    Used to group items
+    """
+    COLLECTIBLE = 0
+    INGREDIENT = 1
+    RECIPE = 2
+    HEALTH = 3
+    UTILITY = 4
+    SONG = 5
+    TURTLE = 6
+
+class AquariaItem(Item):
+    """
+    A single item in the Aquaria game.
+    """
+    game: str = "Aquaria"
+    """The name of the game"""
+
+    def __init__(self, name: str, classification: ItemClassification,
+                 code: Optional[int], player: int):
+        """
+        Initialisation of the Item
+        :param name: The name of the item
+        :param classification: If the item is usefull or not
+        :param code: The ID of the item (if None, it is an event)
+        :param player: The ID of the player in the multiworld
+        """
+        super().__init__(name, classification, code, player)
+
+"""Information data for every (not event) item."""
+item_table = {
+    #       name:           ID,    Nb,   Item Type,        Item Group
+    "Anemone": (698000, 1, ItemType.JUNK, ItemGroup.COLLECTIBLE),  # collectible_anemone
```

Comment:
Id recommend to use a custom dataclass or namedtuple for these value
that way you wont have todo `if data[2] == ItemType.JUNK:` but can do like `if data.type == ItemType.JUNK`